components/scream/src/physics/rrtmgp/atmosphere_radiation.cpp

```diff
@@ -270,6 +271,10 @@ void RRTMGPRadiation::initialize_impl(const RunType /* run_type */) {
   gas_concs.init(gas_names_yakl_offset,m_ncol,m_nlay);
   rrtmgp::rrtmgp_initialize(gas_concs);
 
+  // Set property checks for fields in this process
+
```

Comment:
We should add precondition checks for the surface albedos read in from the surface models here (which should be between 0 and 1).

Reply:
There was a problem in the `PropertyCheck` class, which was fixed in #1531 . If your branch does not contain the commits in that branch, you may want to update the branch first. You have to manually merge master anyways, since github reports the PR as not mergeable, due to conflicts.

Reply:
Great you already fixed it, @bartgol . The 2nd bullet of the commit message for #1531 states exactly the problem I am having. I am going to rebase my branch with master.